src/components/Input/index.jsx

```diff
@@ -1,19 +1,15 @@
-import { MagnifyingGlass } from "@phosphor-icons/react";
+/* eslint-disable react/prop-types */
+
 import styles from "./styles.module.css";
 
-export function Input({ ...rest }) {
+export function Input({ borderNone, isNumber, ...rest }) {
   return (
-    <div className={styles["input-wrapper"]}>
-      <button className={styles["button-search"]}>
-        <MagnifyingGlass size={16} />
-      </button>
-
-      <input
-        {...rest}
-        type="text"
-        className={styles["input-header"]}
-        placeholder="Pesquisar produto"
-      />
-    </div>
+    <input
+      {...rest}
+      type={isNumber === "yes" ? "number" : "text"}
```

Comment:
I think this isNumber is unnecessary, as we are passing `...rest` it already receives the type as a property, so it would be interesting on the page to just pass the type `number` or `text`.

```
<Input 
   type="number"
/>

<Input 
   type="text"
/>
```